.ci/deps.node_js.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e -x
+
+# NPM commands
+ALEX=$(which alex || true)
+# Delete 'alex' if it is not in a node_modules directory,
+# which means it is ghc-alex.
+if [[ -n "$ALEX" && "${ALEX/node_modules/}" == "${ALEX}" ]]; then
+  echo "Removing $ALEX"
+  sudo rm -rf $ALEX
+fi
+
+if [ -z "$TRAVIS_LANGUAGE" ]; then
```

Comment:
not sure what this was about

Reply:
It is unneeded.  If I recall, it was so that a generic job, or a non travis platform, could run the script to do all of the node-stuff.